packs/data/feat-effects.db/effect-ghost-wrangler.json

```diff
@@ -0,0 +1,52 @@
+{
+    "_id": "SiegLMJpVOGuoyWJ",
+    "img": "systems/pf2e/icons/spells/charming-touch.webp",
+    "name": "Effect: Ghost Wrangler",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Ghost Wrangler]{Ghost Wrangler}</p>\n<p>Your fists gain the effects of the ghost touch property rune.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "definition": [
+                    {
+                        "or": [
+                            "item:slug:fist",
+                            "item:slug:unarmed-attack"
+                        ]
+                    }
```

Comment:
```suggestion
                    "item:category:unarmed",
                    "item:base:fist"
```

Reply:
One issue here is that this restricts it to just fist strikes, when most people would flavor their unarmed attack as a fist. The feat itself is truly awful regardless, but I doubt they are wanting to restrict it to only monk barbarians.

Reply:
We could have the feat give a fist attack

Reply:
I guess? It's one of the more useless feats so I kind of just want to stop having the PR held up by it.